docs/develop/setup.md

```diff
@@ -115,8 +115,20 @@ This will install the required packages required for the admin app and start the
 
 The admin app will now be accessible on [http://localhost:3000/](http://localhost:3000/)
 
-## Setting up docs
+## Set up docs
+
+Install [mkdocs](https://www.mkdocs.org/user-guide/installation/).
```

Comment:
And also activate the environment in which you installed it.

Reply:
Ah right. Added!